modules/installation-osp-converting-ignition-resources.adoc

```diff
@@ -0,0 +1,111 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-user.adoc
+
+[id="installation-osp-converting-ignition-resources_{context}"]
+= Preparing the bootstrap Ignition file
+
+The bootstrap Ignition file that the installation program generates is too large to pass to {rh-openstack} directly.
+
+First, upload the bootstrap Ignition configuration file. Then, create a small Ignition file ("shim") that downloads the generated ignition file to Nova as user data.
+
+.Prerequisites
+
+* The bootstrap Ignition file that the installation program generates (`bootstrap.ign`)
+* The infrastructure ID from the installation program's metadata file is set as an environment variable (`$INFRA_ID`)
```

Comment:
see common.yaml

Reply:
Because this procedure precedes the use of any playbooks, should I keep this as it is? It looks like the script in this module requires INFRA_ID. @pierreprinetti

Reply:
you are absolutely right, we use it in the Python script down there. Let's keep it! Good catch.